server.js

```diff
@@ -37,6 +38,9 @@ const queue = new AwaitQueue()
 // @type {Map<Number, Room>}
 const rooms = new Map()
 
+// Maps of eventNotificationUri
+const notificationsUri = new Map()
+
```

Comment:
Maybe not for this PR, but it would be good to start using a Redis DB or something in order to keep track of rooms in a multi-instance environment

Reply:
We can add it without any problem, along with a pub/sub notification logic so that the instances are informed in the first stage. The question that arises is how the routing of traffic (streaming and data) would work between two clients connected to different instances. Theoretically, the instances should create a router when receiving a connection request to that room and link the producers between both. Mediasoup has this option to manage routers, but we need to develop the idea of how to implement this.